### DISCUSSION TITLE
Krokeit unable to delete on Windows 11 - Czkawka throws os error 5?

### DISCUSSION THREAD

**Discussion by adz7** [qarmin/czkawka#1343] [General]
Just logged this bug, anybody able to help?  I've tried everything I can think of, wonder if there is some Windows permission I'm missing somewhere?

Krokeit 7.0 finds thousands of duplicate files, hit select all, select Delete and it results in "failed to remove 4698 items".
Tried Czkawka 6.1.0 same result however at least it provides slightly more info "reason ⁨Access is denied. (os error 5)⁩"

I have checked the following, all drives have current user access set under Security options and have even set the current user as Owner.  Current user is part of Administrative group anyway.
I have run both exe's by right-click 'Run as Admin'.
I have disabled Windows Security, all anti-virus and app protection disabled, even added exceptions for both applications.

Still achieved same result.

**Terminal output** (optional):
Not using Terminal Windows GUI.

I can't locate any error log or debug, Krokiet didn't even throw an error code.

- Czkawka/Krokiet version:  Krokeit 7.0.0 & Czkawa 6.1.0
- OS version: Windows 11 Home - Version 23H2 - Build 22631.4169
- Installation method: Downloaded latest release GUI

**Comment by deadpool2715** [comment]
Check if the directory is read only, I had this issue with the same error message on a folder containing desktop backgrounds from a PC backup and that was the problem

**Reply by adz7** [reply]
Directory not read only and was causing issues for 3 different HDD.  I ended up making a parent New Folder in each drive, cut and pasted all the contents of the hard drive into the New Folder, had to rescan everything but it successfully detected duplicates and deleted them no problem.  So definitely some sort of bug, also Krokeit needs to provide more information other than a "failed to delete" message..

**Comment by AlexioXela** [comment]
I'm having this same issue on Windows 10 with one particular folder. I moved the items from that folder into a new folder, and the new folder has the same issues. Krokiet just can't delete these files, even though I can delete them manually and the directory is not read only.

Edit: Okay, while the _directory_ wasn't read-only, apparently the _files_ were. Removing the read-only attribute fixed it.
